Tests/test_SearchIO_blast_text.py

```diff
@@ -7,14 +7,20 @@
 
 import os
 import unittest
+import warnings
 
 from Bio.SearchIO import parse
-
+from Bio import BiopythonWarning
 
 # test case files are in the Blast directory
 TEST_DIR = 'Blast'
 FMT = 'blast-text'
 
+# This prevents the NCBIStandalone usage warning from
+# printing to screen when running the test suite
+warnings.filterwarnings('ignore', r'Parsing BLAST plain text output '
```

Comment:
It reproduces with pypy locally, I'll investigate a fix. I expected that turning the warnings back on with `warnings.simplefilter("error", BiopythonWarning)` inside the `with` would be enough - but even `warnings.resetwarnings()` doesn't work.

If I can't figure out how to both hide the warning during the test suite and test that the warning is generated by the end of today, I'll just remove the test. It seems mostly unnecessary. My primary goal was to update the test code so that it didn't print the warnings to the screen.